cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
I understand the reasoning behind using Region, Im just not a fan of '_properties_form = prop_region` and subsequently `return self._properties_form.properties_form`... :/